libs/deal-ticket/src/deal-ticket.spec.tsx

```diff
@@ -83,7 +115,7 @@ it('Deal ticket defaults', () => {
 });
 
 it('Can edit deal ticket', () => {
```

Comment:
chore (non-blocking): would be good to add a test here for fraction order sizes and check everything is coming through correctly.